packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,170 @@
+# Nimble Accordion
+
+## Overview
+
+The `nimble-accordion` is a vertical stack of interactive headings. The headings are controls that can be used to reveal or hide content in specific sections. The Nimble accordion is based upon [FAST's accordion component](https://github.com/microsoft/fast/tree/archives/fast-element-1/packages/web-components/fast-foundation/src/accordion).
+
+### Background
+
+[Nimble issue #533: nimble-accordion Component](https://github.com/ni/nimble/issues/533)
+
+[Visual Design](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design&t=DxDRlqT7MeCPLOxi-0)
+
+Visual Design Comments:
+
+Brandon's Figma spec has designs for two accordion item heights, 24px and 32px. In previous cases like the nimble button, we've only implemented one height (in that case only the 32px button was implemented). Do we want to continue doing this, and if so, which height should be implemented?
+
+Brandon's Figma spec also only shows designs for one accordion item at a time. How should accordion items in each appearance type be separated from each other (ex. a line between accordion items, transparent space between accordion items, etc.)?
+
+The Figma design does not have a design for the "error" focusVisible state. This might be confusing, as keyboard users who tab to the error accordion will suddenly see the red border and exclamation mark icon disappear in the focusVisible state.
+
+When should each `appearance` type be used? Will there be specific guidance on this, and if so, will it be similar to the guidance for other Nimble components that use the same `appearance` types (ex. the Nimble-Button)?
+
+## Design
+
+Accordion group, appearance ghost with an error-visible accordion item
+
+```
+<nimble-accordion appearance="ghost">
+    <nimble-accordion-item>
+        <span slot="heading">Accordion one</span>
+            <div slot="default">
+                <nimble-checkbox></nimble-checkbox>
+                <span>Accordion one content</span>
+            </div>
+    </nimble-accordion-item>
+    <nimble-accordion-item expanded>
+        <span slot="heading">Accordion two</span>
+            <div slot="default">
+                <nimble-checkbox></nimble-checkbox>
+                <span>Accordion two content</span>
+            </div>
```

Comment:
Can you double check the indentation of the snippets? This makes it look like the stuff in the default slot is a child of the header but they are siblings.

You can use something like [prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeKBLAtgIwDZwC0AhmJAE4AmG0ABMQA4NzHnFRhwC8AOiPhDABrPgD4eUWlNrps+IqQrVohDPCzjJ07amoA3WgGc8EGLxAALFtSgBzMQEEyEKjUnQ4qAPT7N2nZi4BIRgVsI4EAAeot6B8iFhQhHREv7STkputB60kLAIMKnSsXLBii7KUKrqfsVxZc6uKmpwGkVpuhgGxqbmVsQ29iCiGRVZMADuEN6+7f6yQUQ4AK4wMNCiACpwkTC0AEKr61Ali4Qraxtz2iONldlQcLnQ8LBzp-HlTVUtbVpSC0+dzc1VatQ6+iMJjMfH6g0cwLoMAs5Dgnh8XXB83qRFCcHCURiXhxCXxSUJ1zqpQUiKgWLSAJJX0qoL+DPZMkhPRhlmsGDsYgAcnBDPBKLRRt9aAB5R4zTGUjm0YWiuDiyX3WVPADCLwKivmxOpJFprPpDMBDUyzRqBvZnW60L6fIFwxVYoltNomym8r05qV7rVnutkh9EFouvyb3+HI+VrGNrBBvjNNDWNTJtDZveTNNvwDnK6UN6sJdQ1uodoADMIMtyH7C5alkcNttdgdWycjWcLsdCxqsjk8q9CrHM8yQQWJBPaZoQAAaEAQBgwNyGZCgVjkCATAAKrAQG5QxDwE2IAE8N0ucGxhHAYABlYhYOAAGX5cGQ1dPhjgN7vIQH0fBhSH5WxkBgchln-EA-ywDBIOg2DDHAggAEVllML8kB-PA-yXAArQxIkfNC4Ew7Dv1-WCAEcsPgPcdwYY8QGIQxCEeNU1UXEAoOIDA8HA3UsCwYhkDYvA8F41C7AIBw1nIDALjgPc4HID9Hmo-DYIsGAsDwAB1CwWkMUDOEfI81C6NQLwksBDGvEA9BggBJKBKAKR8wCU1cHA8x8YAvAhtIIkAGB3P9DLYBgJIikV1L0L8l35P9yBgJjiFsMTQtg0DyDSiS9IM3iIv5GBDIwShkWQAAOAAGJdUXojBUUy7LxNwmilxgYgcEq6qLGQAAWJdlj-TY+uPPCwtaHA1U8yg33YWxliyuAADEXDEtZwIk4hVggEAAF9jqAA) to help write the snippets. It'll help find other problems like mismatched tags, etc.